From 8dd37fbc6b9591e921c41dbde8e2187eeafee68d Mon Sep 17 00:00:00 2001
From: ahs <amrita.h.s@intel.com>
Date: Tue, 17 Mar 2020 18:49:46 +0530
Subject: [PATCH] Update manifest.xml

1. Update sepolicy version in manifest.xml
2. delete wifi

Tracked-On:
Signed-off-by: ahs <amrita.h.s@intel.com>
---
 groups/device-specific/caas/manifest.xml      | 30 -------------------
 .../device-specific/celadon_ivi/manifest.xml  |  3 --
 .../celadon_tablet/manifest.xml               |  3 --
 3 files changed, 36 deletions(-)

diff --git a/groups/device-specific/caas/manifest.xml b/groups/device-specific/caas/manifest.xml
index 0b37255..72ec3c9 100644
--- a/groups/device-specific/caas/manifest.xml
+++ b/groups/device-specific/caas/manifest.xml
@@ -151,33 +151,6 @@
             <instance>default</instance>
         </interface>
     </hal>
-    <hal format="hidl">
-        <name>android.hardware.wifi.supplicant</name>
-        <transport>hwbinder</transport>
-        <version>1.1</version>
-        <interface>
-            <name>ISupplicant</name>
-            <instance>default</instance>
-        </interface>
-    </hal>
-    <hal format="hidl">
-        <name>android.hardware.wifi</name>
-        <transport>hwbinder</transport>
-        <version>1.3</version>
-        <interface>
-            <name>IWifi</name>
-            <instance>default</instance>
-        </interface>
-    </hal>
-    <hal format="hidl">
-        <name>android.hardware.wifi.hostapd</name>
-        <transport>hwbinder</transport>
-        <version>1.1</version>
-        <interface>
-            <name>IHostapd</name>
-            <instance>default</instance>
-        </interface>
-    </hal>
     <hal format="hidl">
         <name>android.hardware.media.omx</name>
         <transport>hwbinder</transport>
@@ -302,7 +275,4 @@
         </interface>
     </hal>
 
-    <sepolicy>
-        <version>27.0</version>
-    </sepolicy>
 </manifest>
diff --git a/groups/device-specific/celadon_ivi/manifest.xml b/groups/device-specific/celadon_ivi/manifest.xml
index 49f50b8..d115895 100644
--- a/groups/device-specific/celadon_ivi/manifest.xml
+++ b/groups/device-specific/celadon_ivi/manifest.xml
@@ -281,7 +281,4 @@
             <instance>gpgppu</instance>
         </interface>
     </hal>
-    <sepolicy>
-        <version>27.0</version>
-    </sepolicy>
 </manifest>
diff --git a/groups/device-specific/celadon_tablet/manifest.xml b/groups/device-specific/celadon_tablet/manifest.xml
index 53fe0ed..5971c01 100644
--- a/groups/device-specific/celadon_tablet/manifest.xml
+++ b/groups/device-specific/celadon_tablet/manifest.xml
@@ -282,7 +282,4 @@
             <instance>gpgpu</instance>
         </interface>
     </hal>
-    <sepolicy>
-        <version>27.0</version>
-    </sepolicy>
 </manifest>
-- 
2.17.1

